README.md

```diff
@@ -15,25 +15,39 @@ You will then get access to a client ID and redirect URI you can use for your iO
 
 You can also set a "fallback URL" where users will be redirected if they don't have your app installed.
 
+### Configure your iOS app
 Once you have the custom redirect URL, you can add it to your application as a custom URL scheme.
-If you are not sure how to do so, check out this tutorial: https://dev.twitter.com/cards/mobile/url-schemes
+If you are not sure how to do so, check out this [tutorial](https://dev.twitter.com/cards/mobile/url-schemes).
 
-Once you have the redirect URI setup, go the `UIViewController` where you plan to handle login success/failure, and call `startWithClientId:` as follows:
+To redirect directly to the Clever app, you need to add `com.clever` to your LSApplicationQueriesSchemes in your Info.plist.
```

Comment:
Since this is default behavior, should be stronger with our wording? e.g. Once you have the redirect URI setup, you will need to add `com.clever` ...